.gitignore

```diff
@@ -33,3 +33,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+.env
```

Comment:
Bdw, there is no error here but FYI you can use .env.local file in Next.js application for your local development